Cython/Plex/__init__.py

```diff
@@ -3,7 +3,6 @@
 #   Python Lexical Analyser
 #
 #=======================================================================
```

Comment:
I think it's good to switch comment headers to docstrings, but I think this should be done everywhere.

Reply:
Will make sure to do that throughout Plex now.  Other directories will do in separate PRs.